4_create_test_app_project.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash 
+set -eou pipefail
+
+. ./utils.sh
+
+announce "Creating Test App project."
+
+if has_project "$TEST_APP_PROJECT_NAME"; then
+  echo "Project '$TEST_APP_PROJECT_NAME' exists, not going to create it."
+else
+  echo "Creating '$TEST_APP_PROJECT_NAME' project."
+  oc new-project $TEST_APP_PROJECT_NAME
+
+  # Must run as root to write cert keys to disk.
+  oc adm policy add-scc-to-user anyuid -z default
```

Comment:
lets add a TODO to make sure this doesn't get swept under the rug. overprivileged service account!

Reply:
Fixed.